libipl/p10/ipl0.C

```diff
@@ -387,9 +388,20 @@ static void process_guard_records()
 		return;
 	}
 
-	openpower::guard::libguard_init(false);
+	openpower::guard::GuardRecords records;
+	try {
+		openpower::guard::libguard_init(false);
+	} catch (const openpower::guard::exception::GuardException &ex) {
+		// capture the exception to use it in the PEL
+		// But continue to boot
+		ipl_log(IPL_ERROR,
+			"Caught the exception: %s and continuing to boot",
+			ex.what());
+		ipl_error_callback(IPL_ERR_INVALID_GUARD_FILE);
+		return;
+	}
+	records = openpower::guard::getAll();
```

Comment:
auto records = openpower::guard::getAll();

did not see this

Reply:
done